e2e/cypress/integration/toast/toast_spec.js

```diff
@@ -263,6 +265,29 @@ describe('toasts', () => {
             cy.get('div.toast').should('be.visible').contains('Viewing message history');
         });
     });
+
+    it('MM-T1788 Toast count', () => {
+        // # Visit other channel
+        cy.get(`#sidebarItem_${otherChannel.name}`).should('be.visible').click();
+
+        // # Add 25 posts to create enough space from bottom for showing archive toast
+        for (let index = 0; index < 25; index++) {
+            cy.postMessageAs({sender: otherUser, message: `This is an old message [${index}]`, channelId: townsquareChannelId});
+        }
+
+        visitTownSquareAndWaitForPageToLoad();
+
+        // * Toast should be present
+        cy.get('div.toast').should('be.visible').contains('25 new messages');
+
+        // # Add 25 posts to create enough space from bottom for showing archive toast
+        for (let index = 0; index < 25; index++) {
+            cy.postMessageAs({sender: otherUser, message: `This is an old message [${index}]`, channelId: townsquareChannelId});
+        }
+
+        // * Toast should be present and contains 50 new messages
+        cy.get('div.toast').should('be.visible').contains('50 new messages');
```

Comment:
Given the expectation of "...should listen to the websocket for new messages coming in, updating the counter", I suggest to verify number of messages as it comes in, like:

```javascript
        const initialCount = 25;
        ...
        Cypress._.times(5, (num) => {
            cy.postMessageAs({sender: otherUser, message: `New message [${num}]`, channelId: townsquareChannelId});

            // * Toast should be present and the number of messages increment as it comes in
            cy.get('div.toast').should('be.visible').contains(`${initialCount + num + 1} new messages`);
        });
```

Reply:
> @openmohan Looking good. Please fix merge conflict then we can merge.

Thanks for the quick review @saturninoabril . I have resolved the conflict.